01_intro-r.qmd

```diff
@@ -1,24 +1,34 @@
 ---
 title: "Introduction to R"
-subtitle: "Data Science for Public Policy"
-author: "Aaron R. Williams and Alena Stern - Georgetown University"
 format: 
   html:
     toc: true
     code-line-numbers: true
+    embed-resources: true
 editor_options: 
   chunk_output_type: console
 ---
 
-# Six Principles for Data Analysis
 
-## 1) Accuracy
+```{r}
+#| echo: false
+
+exercise_number <- 1
+
+```
+
+In this chapter, we introduce 6 key concepts for data analysis. We subsequently describe the R programming language and RStudio, its most popular Integrated Development Environment. Finally, we  share key foundational concepts when programming with R. 
+
+
+## Six Principles for Data Analysis
+
+### Accuracy
 
 Deliberate steps *should* be taken to minimize the chance of making an error and maximize the chance of catching errors when errors inevitably occur.
```

Comment:
Deliberate steps *should* be taken to minimize the chance of making an error and maximize the chance of catching errors when errors inevitably occur[^fallibility].

[^fallibility]: [@eubank] influenced our thinking on this issue and [Embrace Your Fallibility: Thoughts on Code Integrity](http://www.nickeubank.com/wp-content/uploads/2016/06/Eubank_EmbraceYourFallibility.pdf) provides proactive steps for writing high-quality code using defensive programming.